### PR TITLE
add Scrutinizer badge

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
I took the liberty to also add the webhook to this repo.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Provides a PHP CodeSniffer ruleset for the MO4 coding standard
 
 [![Build Status](https://travis-ci.org/mayflower/mo4-coding-standard.svg?branch=master)](https://travis-ci.org/mayflower/mo4-coding-standard)
 [![Build Status](https://codecov.io/gh/mayflower/mo4-coding-standard/branch/master/graph/badge.svg)](https://codecov.io/gh/mayflower/mo4-coding-standard/branch/master/)
+[![Scrutinizer Quality Level](https://scrutinizer-ci.com/g/mayflower/mo4-coding-standard/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/mayflower/mo4-coding-standard)
```
Comment (xalopp) [inline README.md]: I recommend to deactivated checking of the `MO4/Tests/*/*.inc` files. They are often broken on purpose 😸

**Comment by xalopp** [comment]
NEIẞ catch:

`UnnecessaryNamespaceUsageSniff::getNamespace()` has wrong return type `array` instead of `string`.
Would be great if you fix that ;-)

**Comment by mmoll** [comment]
done.

**Pull request merged**